sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -33,7 +34,7 @@
 
 
 ###############################################################################
-###########################   Creating Surfaces   #############################
+#                            Creating Surfaces
 ###############################################################################
```

Comment:
Yeah I recognize that this is merely cosmetic and not a crucial piece of this PR. I simply changed them to this style to make it easier to pick out the text in the header rather than hide it with "#" symbols. Perhaps we should make a "style guide" for comments in our sample input files, I'm certainly open to suggestions.